README.md

```diff
@@ -108,7 +108,7 @@ client, err = azkustodata.New(kustoConnectionString)
 #### Using a user-assigned managed identity
 
 ```go
-kustoConnectionString := kustoConnectionStringBuilder.WithUserManagedIdentity(clientID)
+kustoConnectionString := kustoConnectionStringBuilder.WithUserAssignedIdentityClientId(clientID)

```

Comment:
Please do not include code changes in version bump PRs

Reply:
The code change is removing a deprecated API, but I can do this in another PR